libs/core/src/components/calendar/calendar.test.ts

```diff
@@ -410,14 +410,21 @@ describe('<gds-calendar>', () => {
     })
   })
 
+  // Disable for 2023
   describe('Accessibility', () => {
     it('is accessible', async () => {
+      console.warn('Accessibility test skipped')
```

Comment:
It's not skipped, just altered a bit :D Lets talk with jocke about this one when he comes back.

Reply:
Good catch, removed the comment!